### PR TITLE
Switch to basic auth

### PR DESCRIPTION
**Pull request by jbaxter-va**
switch to use basic auth and update how we build the branch name
@vendasta/sre @ChrisPenner

### CODE REVIEW

File: cmd/mapper.go [hunk 1]
```diff
@@ -470,9 +485,10 @@ func cloneRepo(repoName string, dest string) (*git.Repository, error) {
 	fmt.Printf("%s: 🧘‍♂️ Cloning (this could take a while...)\n", repoName)
 	githubRepoURL := fmt.Sprintf("https://github.com/%s/%s", org, repoName)
 
+	fmt.Println(githubRepoURL)
```
Comment (dwalker-va) [inline cmd/mapper.go:488]: can remove or reformat the previous printf to include this

File: cmd/mapper.go [hunk 2]
```diff
@@ -67,9 +71,12 @@ func init() {
 	rootCmd.Flags().StringVarP(&title, "title", "t", "", "Title of the PR")
 	rootCmd.Flags().StringVarP(&description, "description", "d", "", "Description of the PR")
 
-	defaultRSAKeyFile := filepath.Join(homeDir, ".ssh", "id_rsa")
-	rootCmd.Flags().StringVar(&rsaKeyFile, "rsa-key-file", defaultRSAKeyFile, "(optional) The location of an rsa key with github permissions")
-	rootCmd.Flags().StringVar(&rsaKeyPassword, "rsa-key-password", "", "(optional) The password for your ssh key if you have one configured")
+	//defaultRSAKeyFile := filepath.Join(homeDir, ".ssh", "id_rsa")
```
Comment (dwalker-va) [inline cmd/mapper.go:74]: don't need to comment these out if you're supporting both below

**Review by dwalker-va — APPROVED** [review]
working with basic auth i assume?

File: cmd/mapper.go (hunk 1 above)
Comment (jbaxter-va) [reply cmd/mapper.go:488]: that can go

**Comment by jbaxter-va** [comment]
> working with basic auth i assume?

yup works great!

**Pull request merged**